infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/KubernetesSecretApplier.java

```diff
@@ -59,7 +59,7 @@ public abstract void applySecret(E env, RuntimeIdentity runtimeIdentity, Secret
         return env.getDevfile()
             .getComponents()
             .stream()
-            .filter(c -> c.getAlias().equals(componentName))
+            .filter(c -> componentName.equals(c.getAlias()))
```

Comment:
Is the component name guaranteed to be non-null?

Reply:
line 58:      if (componentName != null) {

Reply:
:scream: I'm blind, I'm blind!